fusor-ember-cli/app/models/discovered-host.js

```diff
@@ -18,12 +18,6 @@ export default DS.Model.extend({
   subnet_to_s: DS.attr('string'),
   is_virtual: DS.attr('boolean'),
 
-  // relationship to Engine
-  deployment: DS.belongsTo('deployment', {inverse: 'discovered_host'}),
```

Comment:
For some strange reason, having the definition on the other side in 'discovered-host' was the culprit for the bug. Removing this fixed it.